panel/browser-panel-macos.m

```diff
@@ -0,0 +1,15 @@
+#import <AppKit/AppKit.h>
+
+void RemoveNSViewFromSuperview(void *view)
```

Comment:
Usually it should be possible to cast the result of `GetWindowHandle()` directly to `NSView *`. And ObjC classes (even empty ones) can always be used for message passing, as even `nil` (but not `NULL`) accepts those.

So just checking for a null pointer should be sufficient.

Reply:
> Usually it should be possible to cast the result of GetWindowHandle() directly to NSView *

It should be, but not in C++, right? Or is there a way to (opaquely) use Objective-C classes in (normal) C++?
My idea here was to keep Objective-C out of the C++ file, but also to avoid Objective-C++ in `browser-panel-macos.m(m)` (just because I'm personally not a big fan of ObjC++).

> ObjC classes (even empty ones) can always be used for message passing, as even nil (but not NULL) accepts those.
So just checking for a null pointer should be sufficient.

Interesting, that doesn't work for me.
For the original bug, the pointer isn't a nullptr, but its memory is zeroed. Sending the `removeFromSuperview` message on it crashes reliably, e.g. when closing the Twitch login panel. This is where the (existing) `(*((bool *)view))` comes from.
The `(!view)` is my guesswork for the new crash that I cannot reproduce in the first place.